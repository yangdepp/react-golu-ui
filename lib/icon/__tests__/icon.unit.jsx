import renderer from 'react-test-renderer'
import React from 'react'
import Icon from '../icon'

describe('icon', () => {
  it('是个svg', () => {
    const json = renderer.create(<Icon name="alipay"/>).toJSON()
    expect(json).toMatchSnapshot()
  })
})



//  什么是Snapshot（快照）
//  明确知道测试是对的，就运行yarn test -u，保存对的快照到__snapshots__文件夹中
//  每次运行就会对比上一次存下来的快照