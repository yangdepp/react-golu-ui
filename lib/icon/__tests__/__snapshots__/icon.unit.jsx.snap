// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`icon 是个svg 1`] = `
<svg
  className="golu-icon"
>
  <use
    xlinkHref="#alipay"
  />
</svg>
`;
